README.rst

```diff
@@ -24,6 +24,34 @@ Pyccel comes with a selection of **extensions** allowing you to convert calls to
 - mpi4py
 - h5py (not available yet)
 
+Getting Started
+===============
```

Comment:
I'm not sure the ordering of the README is quite right. I suspect the average user will use pip rather than docker, put the pip installation instructions seem to have become lost at the end of the file.

What do you think @yguclu @saidctb ?

Reply:
@EmilyBourne , true, that was one purpose for a few reasons:
1. docker based pyccel is basically a one line install which is way simpler than the other installation methods (requirements being considered)
2. as such, it fits into a getting started which should be one of the first sections

we want users to easily try out pyccel, and if they like it, they can install it the traditional way.

Reply:
Ok. I still think it would be worth having a line at the beginning to say that an installation with pip is possible even if the details are later.

Most people I know who do not work in IT (including me) have never used docker or have never heard of docker. For those people, having instructions which at first glance imply that installation requires docker, may not be the most encouraging thing

Reply:
I understand so I suggest an introductory chapter specifying all the deployment/installation methods available and then followed by the getting started chapter.

Reply:
That sounds good to me :+1:

Reply:
@EmilyBourne done.
can you check?

Reply:
> I'm not sure the ordering of the README is quite right. I suspect the average user will use pip rather than docker, put the pip installation instructions seem to have become lost at the end of the file.
> 
> What do you think @yguclu @saidctb ?

@EmilyBourne: Sorry for the late reply. @noushi: Thanks for your PR!

I am also in the crowd of people that have never used Docker, and do not see any advantage in using it for their daily work. As far as I know, none of my colleagues use it, and I suspect that many of them does not know what it is. I am sure that some people will be happy to see a Docker file available, but I don't know any.

Most likely, people who want to try Pyccel will probably use it to accelerate one file in one of their Python projects, which is installed on their machine together with all the Python packages required by it. So Pyccel will need to work in that exact environment, maybe on macOS or even Windows. This is why Pyccel needs to be platform-independent, able to use multiple compilers, and with a minimal set of package dependencies.